adapters/automatad/params_test.go

```diff
@@ -0,0 +1,43 @@
+package automatad
+
+import (
+	"encoding/json"
+	"testing"
+
+	"github.com/prebid/prebid-server/openrtb_ext"
+)
+
+func TestValidParams(t *testing.T) {
+	validator, err := openrtb_ext.NewBidderParamsValidator("../../static/bidder-params")
+	if err != nil {
+		t.Fatalf("Failed to fetch the json schema. %v", err)
+	}
+
+	for _, p := range validParams {
```

Comment:
Could you give this variable `p` a more descriptive name

Reply:
Renamed `p` to `validParam`